advanced_alchemy/config/common.py

```diff
@@ -181,6 +181,12 @@ class GenericSQLAlchemyConfig(Generic[EngineT, SessionT, SessionMakerT]):
     This is a listener that will automatically save and delete :class:`FileObject <advanced_alchemy.types.file_object.FileObject>` instances when they are saved or deleted.
 
     Disable if you plan to bring your own save/delete mechanism for these columns"""
+    file_object_raise_on_error: bool = True
```

Comment:
Just curious, is there *ever* a situation where you want to not raise in this case?